filter.lua

```diff
@@ -4,10 +4,15 @@
 --
 
 local xdp     = require("xdp")
```

Comment:
we should have a way (perhaps a flag) to support xdp or nf conditionally.. for instance, for openwifi, we won't build luaxdp modules nor ebpf programs..